packages/styles/components/_c.layer.scss

```diff
@@ -39,28 +39,28 @@
     width: 100%;
 
     @include set-from-screen("m") {
-      max-width: 25rem; // 400px
+      max-width: $layer-max-width-m; // 400px
```

Comment:
From my point of view, we don't have to put in variables, as the values are only used once, and the size does not change depending on the preset.

So we could also have done : 
~~max-width: 25rem; // 400px~~ => `max-width: px-to-rem(400);`

But the evolution made works, so that's the most important thing :)

Reply:
Thanks for the feedback I will make the changes before merging :)